### PR TITLE
Update read me for e2e test

### PR DESCRIPTION
**Pull request by fjglira**
#46

### CODE REVIEW

File: tests/e2e/README.md [hunk 1]
```diff
@@ -2,7 +2,126 @@
...
+* Use the `Specify` block to define the test itself. Remember that `Specify` is an alias for `It` and can be used interchangeably. Use them according to the context of the test.
+* Use `Eventually` to wait for a condition to be met. This is useful when you need to wait for a condition to be met before running the test. Remember that each `It` block should have at least one assertion.
+* Use `Expect` to make direct assertions.
+* Use `Success` helper to print Success message in the test output.
+* Use `kubectl` and `helm` utils to make all the necessary operations in the test that are going to be done by a user. This means that the test should simulate the user behavior when using the operator.
+* Use `client` to interact with the Kubernetes API. This is useful when you need to interact with the Kubernetes API directly and not through `kubectl`. This need to be used to make all the assertions in the test, this means that the test should check the state of the resources in the cluster using the Kubernetes API because are operations that are not going to be done by a user.
+
+## Running the test
```
[the remark this answers is not in the document]
Comment (fjglira) [reply tests/e2e/README.md:123]: Ok, fixed the heading levels

**Review by fjglira — COMMENTED** [review]
Updated from review

**Review by FilipB — APPROVED** [review]

**Pull request merged**